jawampa-core/src/main/java/ws/wamp/jawampa/MessageInfo.java

```diff
@@ -0,0 +1,29 @@
+package ws.wamp.jawampa;
+
+public class MessageInfo<T> {
```

Comment:
I think MessageInfo is a little bit too generic. I would suggest something like EventInfo or EventDetails, because it will only be used for Events and not other Messages.